core/src/mindustry/ui/fragments/PlayerListFragment.java

```diff
@@ -200,7 +200,7 @@ public void draw(){
 
                 button.button(Icon.hammer, ustyle,
                     () -> ui.showConfirm("@confirm", Core.bundle.format("confirmvotekick",  user.name()),
-                    () -> Call.sendChatMessage("/votekick " + user.name())))
+                    () -> Call.sendChatMessage("/votekick #" + user.id)))
```

Comment:
This is only telling the user to type in `/votekick #69`.

Does the command `/votekick #69` actually work though?

~~(I probably shouldn't be asking this, I don't do any sort of server stuff.)~~

Reply:
I would hope so, considering he said it was tested.